Jenkinsfile

```diff
@@ -107,45 +99,88 @@ pipeline {
                     colourText("info","Generating reports for tests")
                     //   junit '**/target/test-reports/*.xml'
 
-                    step([$class: 'CoberturaPublisher', coberturaReportFile: '**/target/scala-2.11/coverage-report/*.xml'])
-                    step([$class: 'CheckStylePublisher', pattern: 'target/scalastyle-result.xml, target/scala-2.11/scapegoat-report/scapegoat-scalastyle.xml'])
+                    // removed subfolder scala-2.11/ from target path
+                    step([$class: 'CoberturaPublisher', coberturaReportFile: '**/target/coverage-report/*.xml'])
+                    step([$class: 'CheckStylePublisher', pattern: '**/target/code-quality/style/*scalastyle*.xml'])
                 }
                 failure {
                     colourText("warn","Failed to retrieve reports.")
                 }
             }
         }
 
-
-        // bundle all libs and dependencies
-        stage ('Bundle') {
+        stage('Build'){
             agent any
-            when {
-                anyOf {
-                    branch BRANCH_DEV
-                    branch BRANCH_TEST
-                    branch BRANCH_PROD
+            steps {
+                dir('gitlab') {
+                    git(url: "$GITLAB_URL/StatBusReg/${MODULE_NAME}-api.git", credentialsId: GITLAB_CREDS, branch: "${BRANCH_DEV}")
+                }
+                // Replace fake VAT/PAYE data with real data
+                sh 'rm -rf conf/sample/201706/vat_data.csv'
+                sh 'rm -rf conf/sample/201706/paye_data.csv'
+                sh 'cp gitlab/dev/data/sbr-2500-ent-vat-data.csv conf/sample/201706/vat_data.csv'
+                sh 'cp gitlab/dev/data/sbr-2500-ent-paye-data.csv conf/sample/201706/paye_data.csv'
```

Comment:
Where is this actuall? used in sbr-api, is not used in the admin data APIs instead?